newton/_src/viewer/viewer_viser.py

```
@@ -1313,6 +1333,86 @@ def log_points(
+
+        if hidden:
+            if name in self._scene_handles:
+                self._scene_handles[name].visible = False
+            return
+
+        if gaussian is None or gaussian.count == 0:
+            if name in self._scene_handles:
+                try:
+                    self._scene_handles[name].remove()
+                except Exception:
+                    pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)